var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -61,9 +61,9 @@ class Mpich(AutotoolsPackage, CudaPackage, ROCmPackage):
     variant("wrapperrpath", default=True, description="Enable wrapper rpath")
     variant(
         "pmi",
-        default="pmi",
+        default="default",
```

Comment:
Is `default` "consistent" across similar systems, or is it based on feature detection on the host where configure happens?

Reply:
Consistent across systems. We do not do any feature detection that would influence the default selection.

Reply:
Thanks, then I think it's safe to use it as a default value for the variant.